sycl/test-e2e/lit.cfg.py

```diff
@@ -646,6 +646,10 @@ def open_check_file(file_name):
 for sycl_device in config.sycl_devices:
     env = copy.copy(llvm_config.config.environment)
     env["ONEAPI_DEVICE_SELECTOR"] = sycl_device
+    env["ZES_ENABLE_SYSMAN"] = "1"
```

Comment:
Since https://github.com/oneapi-src/unified-runtime/commit/27ad3f7d95a4e78e10c31df0e5c73afe549d1449 sysman is expected to be enabled by default. So, I believe this change is redundant now.

Reply:
Oh, thanks for letting me know 👍